scripts/docs/generate_asset_documentation.py

```diff
@@ -14,6 +14,7 @@
 
 SUPPORTED_ASSET_TYPES = [AssetType.ENVIRONMENT, AssetType.COMPONENT, AssetType.MODEL, AssetType.DATA]
 DEFAULT_CATEGORY = "Uncategorized"
+ALREADY_PLURALIZED_TYPES = ["data"]
```

Comment:
Consider making a dict of type to plural values so they can all be handled consistently.